demosplan/DemosPlanCoreBundle/Logic/User/OrgaHandler.php

```diff
@@ -159,7 +159,6 @@ public function getWritableAttributes(): array
             'ccEmail2',
             'copySpec',
             'cssvars',
-            'email2',
```

Comment:
Why do you think, this is no longer used? I still see the property on the Orga entity which to me suggests, that the attribute should also still be writable.

Reply:
In `DpOrganisationFormFields.vue:407-584` used **email2** as a condition and i should remove it to not show email field. (base on Florian's Comment )

Reply:
what about [3250e47](https://github.com/demos-europe/demosplan-core/pull/1940/commits/3250e47382beef4c3f55e0e9e394c0196418d0e3)